web_x2many_delete_all/static/src/js/web_x2many_delete_all.js

```diff
@@ -21,7 +21,9 @@ odoo.define('web_x2many_delete_all.Mixin', function (require) {
         _renderDeleteAll: function (tag) {
             var $icon = $('<button>', {class: 'fa fa-trash-o o_list_record_delete_all_btn', name: 'delete all',
                 'aria-label': _t('Delete All')});
-            return $('<th>', {class: 'o_list_record_delete_all'}).append($icon);
+            return $('<' + tag + ' width="1">')
```

Comment:
The `width` attr was long ago deprecated in favor of CSS styling. Better, add the desired width in the Less code and remove it from here. Thanks!

Reply:
Removed !! :+1: